metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -54,6 +63,7 @@ class GlueSourceConfig(AwsSourceConfig):
     ignore_unsupported_connectors: Optional[bool] = True
     emit_s3_lineage: bool = False
     glue_s3_lineage_direction: str = "upstream"
+    domain: Dict[str, AllowDenyPattern] = dict()
```

Comment:
name this `domain_pattern`?

Reply:
This is the name in the other sources and I wanted to keep it consistent.